### DISCUSSION TITLE
Ability to copy source as markdown

### DISCUSSION THREAD

**Discussion by href** [outline/outline#6254] [Ideas]
It is already possible to download a page as Markdown, it would be nice if we could have a way to directly copy it into the clipboard.

**Comment by tommoor** [comment]
This has been added as an action to the [command menu](https://docs.getoutline.com/s/guide/doc/navigation-Nxnr2lWbck#h-command-menu) now. It's live on cloud hosted and will be in the next release in around 3 weeks.

<img width="779" alt="image" src="https://github.com/outline/outline/assets/380914/fa4246fd-e9c2-4bab-abf3-3afb52391ce7">

**Reply by href** [reply]
Awesome, thank you! ❤️
